packages/node/test/fixtures/37-vercel-og/package.json

```diff
@@ -1,6 +1,5 @@
 {
   "private": true,
-  "type": "module",
```

Comment:
Why did this need to be removed?

Reply:
The test fails:

> Rejected to value: [Error: Must use import to load ES Module: /home/runner/work/vercel/vercel/packages/node/test/fixtures/37-vercel-og/probe.js]

I think its legit since the code is not actually using `import`